src/app/containers/MediaPlayer/Metadata.jsx

```diff
@@ -14,4 +15,10 @@ const Metadata = aresMediaBlock => {
   );
 };
 
+Metadata.propTypes = {
+  aresMediaBlock: shape({
+    model: shape({}),
```

Comment:
Presumably this property is required? Otherwise what is the point of specifying it?